Sources/Models/AvatarPosition.swift

```diff
@@ -46,12 +46,6 @@ public struct AvatarPosition {
     /// An enum representing the verical alignment for an `AvatarView`.
     public enum Vertical {
         
-        /// Aligns the `AvatarView`'s top edge to the cell's top edge.
-        case cellTop
```

Comment:
I was thinking about leaving these in the case of a user not using the `cellTopLabel`. If they mess up and use `.cellTop` with a `cellTopLabel` then this is a user error. 

Let me know what you think?

Reply:
I prefer to remove. I don't like overlap of views. `AvatarView` maybe overlap `topMessageLabel` and `topCellLabel`, blended layers, and also misleading of tap gesture handler. More important, I think that seldom people would needs this case. 😂

Reply:
So we should preserve .cellTop case, then, we should let avatar align the top equal to `topCellLabel`'s top or `topMessageLabel`'s top?

Reply:
@zhongwuzw Yeah, I think this is a good idea. We can add back `.cellBottom` to be consistent too.

Reply:
Yes, overlapping views is a common request for many users.

Reply:
@SD10 , so we should make `avatarView`'s top equal to `topMessageLabel`'s top in `.cellTop` case?

Reply:
@zhongwuzw Yeah this is confusing. Should we add a `.messageLabelTop` for this?

Reply:
🤔 , I think we should, most users maybe need `.messageLabelTop`, other than `.cellTop`, I'll add these two cases.